app/Bookstore.Domain/Addresses/AddressService.cs

```diff
@@ -44,6 +49,8 @@ public async Task CreateAddressAsync(CreateAddressDto dto)
             await addressRepository.AddAsync(address);
 
             await addressRepository.SaveChangesAsync();
+
+            logger.LogInformation("Saved a new address for the Customer with Id {id}. Address : {serializedaddress}", address.CustomerId, JsonSerializer.Serialize(address));
```

Comment:
Would it be worth using string interpolation here instead? i.e. 

logger.LogInformation($"Saved a new address for the Customer with Id {address.CustomerId}. Address : {JsonSerializer.Serialize(address)}");